detox/local-cli/test.js

```diff
@@ -125,7 +125,13 @@ module.exports.builder = {
   'device-launch-args': {
     group: 'Execution:',
     describe: 'Custom arguments to pass (through) onto the device (emulator/simulator) binary when launched.'
-  }
+  },
+  'use-custom-logger': {
```

Comment:
@d4vidi , I would rather invert logic, e.g.: `--no-console-patch`.

Reply:
I was actually going back and forth with this, I'll revisit

Reply:
'decided not to change seeing the a `no` prefix is something mocha omits and recognizes as a false. So for example, a `--no-bugs` will end up being passed in as `--bugs false` by mocha. All in all, that differs from Jest and I preferred having them unified. And yes, I'm pretty sure the current `--no-color` arg is broken.